packages/nimble-components/src/select/styles.ts

```diff
@@ -29,6 +30,25 @@ export const styles = css`
          error icon, and dropdown arrow because they are not "interactive" i.e. part of the tab order */ ''
     }
 
+    ::slotted(nimble-list-option-group.show-top-separator)::before,
```

Comment:
Why make `show-top-separator` and `last-visible-option-group` classes while `visually-hidden` is a property/attribute? It seems like those should be consistent. I also don't think we tend to add/remove css classes to elements created by a client app.

Reply:
Transitioned this API to attributes, which there are only two of: `showTopSeparator` and `showBottomSeparator`.